src/main/java/jalau/at18/katas/bankocr/rodrigob/Scanner.java

```diff
@@ -0,0 +1,15 @@
+package jalau.at18.katas.bankocr.rodrigob;
+
+public class Scanner {
+
+    public Scanner() {
+    }
+
+    public char[][] readInput() {
```

Comment:
Is this scanner intended to be re-usable? if so, maybe code in this method should change so it can convert any type of input read